### DISCUSSION TITLE
Plug in to charge and system turns on fans despite being in silent mode

### DISCUSSION THREAD

**Discussion by thduc87** [seerge/g-helper#401] [General]
![image](https://github.com/seerge/g-helper/assets/132545384/b0956dbd-d502-4fb7-85f8-20df60d6f54c)
Hi, i experienced this issue after update to latest version 0.63. When i plug in usb-c to charge then the system turns fans on automatically despite still being in silent mode.

**Comment by thduc87** [comment]
just realized after a brief moment the fans have turned off, is this normal behavior?

**Reply by seerge** [reply]
@thduc87 i have no idea :) it's your laptop, and fans are controlled by bios, not by the app.
